frontend/package.json

```diff
@@ -12,6 +12,7 @@
     "test": "vitest"
   },
   "dependencies": {
+    "@phosphor-icons/vue": "^2.2.1",
```

Comment:
Just wondering: Do we have any decision on icon libraries that we use? When starting to implement Appointment, I added the tabler icons just for convenience to fill in design gaps. So we can totally replace them with design SVGs or icons from another lib.

Reply:
Perfect, thanks for this information! Do we have a ticket for that? I can file one if needed.